tests/testthat/test-linter_tags.R

```diff
@@ -112,27 +112,35 @@ test_that("lintr help files are up to date", {
   # 2. Rscript -e "library(lintr); testthat::test_file('tests/testthat/test-linter_tags.R')"
 
   helper_db_dir <- system.file("help", package = "lintr")
-  skip_if_not(dir.exists(helper_db_dir))
-  skip_if_not(file.exists(file.path(helper_db_dir, "lintr.rdb")))
-  skip_if_not(file.exists(file.path(helper_db_dir, "lintr.rdx")))
+  skip_if_not(
+    dir.exists(helper_db_dir) && all(file.exists(file.path(helper_db_dir, c("lintr.rdb", "lintr.rdx")))),
+    message = "Package help not installed or corrupted"
+  )
 
   help_env <- new.env(parent = topenv())
   lazyLoad(file.path(helper_db_dir, "lintr"), help_env)
 
-  lintr_db <- available_linters(exclude_tags = NULL)
+  lintr_db <- available_linters()
   lintr_db$package <- NULL
   lintr_db$tags <- lapply(lintr_db$tags, sort)
 
-  expect_true(exists("linters", envir = help_env), info = "?linters exist")
-  # objects in help_env are class Rd, see ?as.character.Rd (part of 'tools')
-  linter_help_text <- paste(as.character(help_env$linters), collapse = "")
+  expect_true(exists("linters", envir = help_env), info = "?linters exists")
+
+  # NB: objects in help_env are class Rd, see ?as.character.Rd (part of 'tools')
+  rd_as_string <- function(rd) paste(as.character(rd), collapse = "")
```

Comment:
Nice helper 😃

Reply:
the whole test is still a bit of a mess to understand from scratch, this is a small improvement :)